src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started its work.");
```

Comment:
The message 'Bulldozer started its work.' is not very informative. It could be improved by specifying what kind of work the Bulldozer is doing, for example, 'Bulldozer started its earthmoving work.'